### PR TITLE
MCQ questions can be added & viewed.

### PR DESCRIPTION
**Pull request by vaibhav0320**
Questions with more then 1 option filled will be marked as MCQ automatically.

### CODE REVIEW

**Comment by Niraj-Kamdar** [comment]
put your .vscode/ folder into .gitignore file

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
```
Comment (Niraj-Kamdar) [inline .vscode/settings.json]: this shouldn't be here. put it in .gitignore

File: flaskapp/forms.py [hunk 2]
```diff
@@ -12,3 +12,8 @@ class QuestionForm(FlaskForm):
                               validators=[DataRequired(), NumberRange(1, 100, "Not a valid difficulty")])
     imp = BooleanField('imp')
     submit = SubmitField('submit')
+    option1 = TextAreaField('Option1')
```
Comment (Niraj-Kamdar) [inline flaskapp/forms.py]: It shouldn't be like this there should be two forms one for subjective and one for objective. Make a form with selector to select type of the question then provide form with appropriate type

File: flaskapp/models.py [hunk 3]
```diff
@@ -7,13 +7,23 @@ class Question(db.Model):
     mark = db.Column(db.Integer, nullable=False)
     difficulty = db.Column(db.Integer, nullable=False)
     imp = db.Column(db.Boolean, default=False)
+    option1 = db.Column(db.Text,nullable=True,default=None)  #
```
Comment (Niraj-Kamdar) [inline flaskapp/models.py]: same here two models, use specialization here. mcq_question **is a** question kind of relation. checkout sqlalchemy for this. Maybe you can do this with backref

File: flaskapp/routes.py [hunk 4]
```diff
@@ -26,22 +26,47 @@ def questions():
                            )
 
 
+def check_for_mcq(form):
```
Comment (Niraj-Kamdar) [inline flaskapp/routes.py]: this won't needed when you implement above method

**Comment by vaibhav0320** [comment]
Can you give example of how to write update test?

File: .vscode/settings.json (hunk 1 above)
Comment (vaibhav0320) [reply .vscode/settings.json]: ok

**Pull request closed without merging**